src/symbols/tlaSymbols.ts

```diff
@@ -492,7 +492,7 @@ export class TlaDocumentSymbolsProvider implements vscode.DocumentSymbolProvider
                             const col = parseInt(location.column?.begin || '0') - 1;
                             const endLine = parseInt(location.line?.end || '0') - 1;
                             const endCol = parseInt(location.column?.end || '0') - 1;
-                            const preComments = declNode['pre-comments'] || undefined;
+                            const preComments = declNode['pre-comments'] ? cleanCommentMarkers(declNode['pre-comments']) : undefined;
```

Comment:
I think you could also replace this with `cleanCommentMarkers(declNode['pre-comments'])` as it's checked on line 839.